README.md

```diff
@@ -104,6 +104,21 @@ $ remme account get-balance \
 368440.0
 ```
 
+### Batches
+
+Get batch by its identifier — ``remme batch get-single``:
+
+| Arguments | Type   |  Required | Description                                         |
+| :-------: | :----: | :-------: | --------------------------------------------------- |
+| id        | String |  Yes      | Get batch by its identifier.                                |
+| node-url  | String |  No       | Apply the command to the specified node by its URL. |
+
+```bash
+$ remme batch get-single \
+      --id=ce5d2047f1a34bafbe228fac392573435b7048352b1f013af69e6bbf8fc10e5156915d7ee94f0d4a2fe363e7583f7039f29b1ab50d2e139e0957b5d08740bfd9 \
+      --node-url=node-genesis-testnet.remme.io
```

Comment:
Add example of the response, please. As [here](https://github.com/Remmeauth/remme-core-cli/tree/block-commands#block). Do it also for pull request description.

Reply:
Done.